packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -199,14 +202,39 @@ class RenderAndroidView extends RenderBox with _PlatformViewGestureMixin {
     markNeedsPaint();
   }
 
+  // Sets the offset of the underlaying platform view on the platform side.
+  //
+  // This allows the Android native view to draw the a11y highlights in the same
+  // location on the screen as the platform view widget in the Flutter framework.
+  //
+  // It also allows platform code to obtain the correct position of the Android
+  // native view on the screen
+  void _setOffset() {
+    SchedulerBinding.instance.addPostFrameCallback((_) async {
+      if (!_isDisposed) {
```

Comment:
nits: can break early:
```
if (_isDisposed) {
  return;
}
```

Reply:
I don't think this is necessary. Even if added, it would need to be added in two places.